library/include/modules/Units.h

```diff
@@ -81,6 +81,7 @@ DFHACK_EXPORT bool isFortControlled(df::unit *unit);
 DFHACK_EXPORT bool isOwnCiv(df::unit* unit);
 DFHACK_EXPORT bool isOwnGroup(df::unit* unit);
 DFHACK_EXPORT bool isOwnRace(df::unit* unit);
+DFHACK_EXPORT bool isNobleFromOtherSite(df::unit* unit, bool ignore_own = true);
```

Comment:
functions that are exposed to Lua via WRAPM must not have parameters that default to `true`. The Lua layer only knows what the type signature is -- it doesn't know anything about C++ default parameters. Calling this function from Lua as `dfhack.units.isNobleFromOtherSite(unit)` will result in `ignore_own == false`.